src/test/regress/expected/multi_mx_hide_shard_names.out

```diff
@@ -74,15 +74,14 @@ SELECT * FROM citus_shard_indexes_on_worker WHERE "Schema" = 'mx_hide_shard_name
 -- make sure that pg_class queries do not get blocked on table locks
 begin;
 lock table test_table in access exclusive mode;
-prepare transaction 'take-aggressive-lock';
```

Comment:
Instead of changing this, we should do:

```SQL
BEGIN;
SET LOCAL citus.enable_ddl_propagation TO OFF;
lock table test_table in access exclusive mode;
prepare transaction 'take-aggressive-lock';